scripts/travis/upload-all-docker-images.sh

```diff
@@ -5,13 +5,12 @@
 set -euxf -o pipefail
 
 BRANCH=${BRANCH:?'missing BRANCH env var'}
-DOCKERHUB_LOGIN=${DOCKERHUB_LOGIN:-false}
 
-# Only push images to Docker Hub for master branch or for release tags vM.N.P and when dockerhub login is done
-if [[ ("$BRANCH" == "master" || $BRANCH =~ ^v[0-9]+\.[0-9]+\.[0-9]+$) && "$DOCKERHUB_LOGIN" == "true" ]]; then
-  echo "upload to Docker Hub, BRANCH=$BRANCH"
+# Only push images to dockerhub/quay.io for master branch or for release tags vM.N.P
+if [[ "$BRANCH" == "master" || $BRANCH =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
```

Comment:
seems like every script is doing this check, so could be another candidate to move into upload-to-registry